### ISSUE TITLE
Toggle to reverse state if passed no arguments

### ISSUE THREAD

**Issue by jussi-kalliokoski** [nelsonjchen/fntoggle#1]
Would be nice for implementing a shortcut key for this :)

**Comment by megadr01d** [comment]
Great project. I've been looking for a solution without UI scripting for a long time.

@jussi-kalliokoski I agree that the name "toggle" would imply a single toggle command but having two commands may also have its advantages (to know programatically if it's on or off).
Anyway, I'm using the following script with LaunchBar, you can wrap it in an AppleScript or Application bundle. I'm also using `terminal-notifier` to get notifications. You can install it with Homebrew or remove those lines altogether.

~~~bash
#!/bin/sh

file="$HOME/.fntoggle"

if grep -q "fntoggle on" $file
then
    /usr/local/bin/fntoggle off
    echo "[FunctionKeysDisabled]\nfntoggle off" > $file
    /usr/local/bin/terminal-notifier -title "LaunchBar" -subtitle "Switch Function Keys" -message "Function keys are disabled" -sender "at.obdev.LaunchBar"
else
    /usr/local/bin/fntoggle on
    echo "[FunctionKeysEnabled]\nfntoggle on" > $file
    /usr/local/bin/terminal-notifier -title "LaunchBar" -subtitle "Switch Function Keys" -message "Function keys are enabled" -sender "at.obdev.LaunchBar"
fi
~~~

**Comment by megadr01d** [comment]
@jussi-kalliokoski I haven't tested it but [this fork](https://github.com/mscreations/fntoggle) seems to implement toggle state option just by invoking the binary with no arguments.

**Comment by nelsonjchen** [comment]
Closed by #3
